tests/bin/eslint.js

```diff
@@ -258,15 +258,18 @@ describe("bin/eslint.js", () => {
             it("updates the cache file when the source file is modified", () => {
                 const initialCacheContent = fs.readFileSync(CACHE_PATH, "utf8");
 
-                // Update the file to change its mtime
-                fs.writeFileSync(SOURCE_PATH, fs.readFileSync(SOURCE_PATH, "utf8"));
+                // Update the file to add one blank space
+                fs.writeFileSync(SOURCE_PATH, fs.readFileSync(SOURCE_PATH, "utf8").concat(" "));
```

Comment:
this change is for the changes in `file-entry-cache` [Use content checksum instead of mtime and fsize](https://github.com/royriojas/file-entry-cache/commit/343b340847408afc321d942f90ab0ec5c1d1501b#diff-2ea7ee6033c7ced693a37e8ab9b18151)

Reply:
I have concerns about the performance impact of this. It seems like reading all files from the filesystem would significantly hurt performance when linting with a cache file.

(It also has the effect that we would be reading most files from the filesystem twice when we lint them, so it seems like we could get a significant performance boost by just implementing caching ourselves.)

Reply:
@not-an-aardvark I did realize we merged the content checksum check instead of the mtime/size and was thinking to actually make that optional, to keep the original behavior. That's why that was released as a major version and not a minor one.